hc/front/tests/test_add_channel.py

```diff
@@ -37,5 +36,30 @@ def test_instructions_work(self):
             r = self.client.get(url)
             self.assertContains(r, "Integration Settings", status_code=200)
 
-    ### Test that the team access works
-    ### Test that bad kinds don't work
+
+class TeamAccessTestCase(BaseTestCase):
+    # Test that the team access works
+    def test_team_access_works(self):
+        kinds = ("email", "webhook", "pd", "pushover", "hipchat", "victorops")
+        for kind_name in kinds:
+            self.channel = Channel(user=self.alice, kind=kind_name)
+            self.channel.value = "test that team access works"
+            self.channel.save()
+
+            url = "/integrations/{}/checks/".format(self.channel.code)
+
+            self.client.login(username="bob@example.org", password="password")
+            self.response = self.client.get(url)
+            self.assertEquals(self.response.status_code, 200)
+
+
+# Test that bad kinds don't work//channels that raise Not implemented error
+class BadKindTestCase(BaseTestCase):
+    def test_bad_kind(self):
+        self.client.login(username="alice@example.org", password="password")
+        bad_kinds = ('0798885255', 'twitter', 'instagram', 'whatsapp',
+                     'telegram', 'facebook', 'reddit')
+        for any_kind in bad_kinds:
+            url = "/integrations/add_%s/" % any_kind
+            r = self.client.get(url)
+            assert r.status_code == 404
```

Comment:
Replace this assert statement with its equivalent in Django.

Resource: [Django assertions](https://docs.djangoproject.com/en/1.10/topics/testing/tools/#assertions)